tcms/testcases/models.py

```diff
@@ -339,14 +339,14 @@ def add_cc(self, email_addrs):
     def get_cc_list(self):
         """ Return the whole CC list """
         if not self.cc_list:
-            return []
+            return [] #pylint:disable=Protected-access
         return self.cc_list.split(MultipleEmailField.delimiter)
 
     def remove_cc(self, email_addrs):
         """Remove one or more email addresses from EmailSettings' CC list
 
         If any email_addr is unknown, remove_cc will keep quiet.
-
+#pylint:disable=Protected-access
```

Comment:
This is a comment inside a documentation string for the function `remove_cc`. It doesn't do anything. Remove it.